mule-wrapper-additional-parameters-parser/src/test/java/org/mule/runtime/params/AdditionalJvmParametersTestCase.java

```diff
@@ -216,4 +221,31 @@ public void testAdditionalJvmParameters() throws IOException {
     assertThat(wrapperAdditionalConfFile).hasContent(sb.toString());
   }
 
+  @Test
+  public void testProcessBootstrapProperties() throws IOException {
+    String wrapperPrefix = "wrapper.java.additional.";
+    StringWriter writer = new StringWriter();
+    Properties properties = new Properties();
+    properties.setProperty("wrapper.java.additional.<n1>", "value1");
+    properties.setProperty("wrapper.java.additional.<n1>.suffix", "value1");
+    properties.setProperty("wrapper.java.additional.<n2>", "value2");
+    properties.setProperty("wrapper.java.classpath.<n1>", "classpathValue1");
+    properties.setProperty("other.property", "otherValue");
+
+    AdditionalJvmParameters.processBootstrapProperties(properties, writer);
+
+    List<String> actualList = asList(writer.toString().split("\n"));
+    List<String> list = actualList.stream().filter(s -> s.matches("wrapper\\.java\\.additional.*value1")).collect(toList());
+    assertThat(list.size(), is(2));
```

Comment:
Sorry I missed this one in the previous review. We can also use a better matcher here.